guides/common/modules/ref_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,56 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {EL} 7 to {EL} 8 includes an upgrade from Ansible Engine 2.9 to Ansible Core 2.12.
+
+If you have custom Ansible content such as playbooks, job templates inside REX, roles and collections on disk, and you relied on modules being delivered by the Ansible RPM on {Project}, you have to take additional steps to adapt your Ansible installation or migrate your Ansible content.
+
+Ansible Core removed all non-essential modules shipped in the core installation.
+In terms of FQCN notation `namespace.collection.module`, you can continue to use `ansible.builtin.*`, but everything else is missing in Ansible Core.
+That means you will be no longer able to use non-builtin Ansible modules as you were used to and you have to get them from somewhere else, eventually.
+
+Current options to resolve this situation:
+
+* You can obtain additional community-maintained collections that provide the removed functionality from Ansible Galaxy.
+For more information, see https://docs.ansible.com/ansible/devel/user_guide/collections_using.html#installing-collections[Installing collections] in _Ansible Galaxy Documentation_.
+ifdef::satellite[]
++
+Note that {Team} does not provide support for this content.
+endif::[]
+ifdef::satellite[]
+* If you have a subscription for https://www.ansible.com/products/automation-hub[Red Hat Automation Hub], you can configure your `ansible-galaxy` to talk to Automation Hub server and download content from there.
+That content is supported by {Team}.
+For more information on configuring Automation Hub connection for `ansible-galaxy`, see
+https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html/getting_started_with_automation_hub/proc-configure-automation-hub-server[Configuring Red Hat automation hub as the primary source for content].
+endif::[]
+* You rewrite your templates, roles, etc.
+ifdef::satellite[]
+Note that {Team} does not provide support for content that you maintain yourself.
+endif::[]
+
+[NOTE]
+====
+If you want to download and install Ansible content on {SmartProxy} that does not have a connection to an external Galaxy server, then you must pass the content through {ProjectServer} instead of using the URL of the Galaxy server in the configuration on the {SmartProxy} directly:
```

Comment:
```suggestion
If you want to download and install Ansible content on {SmartProxy} that does not have a connection to an external Ansible Galaxy server, then you must pass the content through {ProjectServer} instead of using the URL of the Ansible Galaxy server in the configuration on the {SmartProxy} directly:
```